pkg/apis/serving/v1alpha1/route_conversion_test.go

```diff
@@ -77,9 +79,20 @@ func TestRouteConversion(t *testing.T) {
 							Percent:      ptr.Int64(100),
 						},
 					}},
-					// TODO(mattmoor): Addressable
-					// TODO(mattmoor): Domain
-					// TODO(mattmoor): DomainInternal
+					Address: &duckv1alpha1.Addressable{
```

Comment:
(maybe a separate diff) given that we have different `Addressable` types for `v1beta1` and `v1` should we have a a `v1alpha1.Addressable` that doesn't contain the `v1beta1` type?

Reply:
Probably would be more clear to not have it nested, but given that this change lives it pkg and is breaking I am not sure it is worth the effort. Once we are sure that no v1alpha1 types have a populated Hostname we should be able to just migrate v1alpha1 Serving types to use v1.Addressable.